File: db4o.cs/src/com/db4o/cs/internal/ClientConnectionEventArgs.java
package com.db4o.cs.internal;

import com.db4o.events.*;

public class ClientConnectionEventArgs extends EventArgs {
	
	private final ClientConnection _connection;

	public ClientConnectionEventArgs(ClientConnection connection) {
	    _connection = connection;
    }
	
	/**
	 * @sharpen.property
	 */
	public ClientConnection connection() {
		return _connection;
	}
}

File: db4o.net/Db4objects.Db4o.CS/Db4objects.Db4o.CS/CS/Internal/ClientConnectionEventArgs.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using System;
using Db4objects.Db4o.CS.Internal;

namespace Db4objects.Db4o.CS.Internal
{
	public class ClientConnectionEventArgs : EventArgs
	{
		private readonly IClientConnection _connection;

		public ClientConnectionEventArgs(IClientConnection connection)
		{
			_connection = connection;
		}

		public virtual IClientConnection Connection
		{
			get
			{
				return _connection;
			}
		}
	}
}
